dashboard/assets/packages/ui/src/components/Nav/Nav.tsx

```diff
@@ -0,0 +1,69 @@
+// SPDX-FileCopyrightText: 2023 Raintank, Inc. dba Grafana Labs
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import React from "react"
+
+import { toClassName } from "utils"
+import { Button } from "components/Button"
+import { Flex } from "components/Flex"
+
+import * as styles from "./Nav.css"
+
+interface Option {
+  title?: string
+  id?: string
+}
+
+interface NavProps {
+  isMobile?: boolean
+  options: Option[]
+  value: number
+  onChange: (idx: number) => void
+}
+
+export function Nav({ isMobile = false, options, value, onChange }: NavProps) {
+  return (
+    <Flex
+      as="nav"
+      gap={2}
+      className={toClassName(
+        styles.nav,
+        styles.navVariant({
+          display: {
+            desktop: isMobile ? "none" : "flex",
+            mobile: isMobile ? "flex" : "none"
+          }
+        })
+      )}>
+      {options.map((option, index) => (
+        <Item
+          key={option.id}
+          aria-controls={`nav-${index}`}
```

Comment:
Can remove the `aria-controls` here as it isn't hooked up correctly and as said for the tab panel, I think isn't the right tool to use in this context.